tooling/SecretManager.VisualStudio/ProjectLocalSecretsManager.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Threading;
+using Task = System.Threading.Tasks.Task;
+
+namespace Microsoft.Extensions.SecretManager.VisualStudio
+{
+    public class ProjectLocalSecretsManager : IVsProjectSecrets, SVsProjectLocalSecrets
```

Comment:
You mentioned to me some concerns about interacting with open files and the RDT. We can talk about this and figure it out tomorrow.